app/samples/sensor/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.20.0)
 
-set(BOARD nucleo_f446re)
```

Comment:
Not really the suggested way of doing things, updated makefile to pass board at build time